cset-workflow/app/build_conda/bin/build_conda_env.sh

```diff
@@ -51,6 +53,7 @@ if [[ $CSET_ENV_USE_MODULES == True ]]; then
       # a module to make conda available. This is to simplify the logic.
       module load "$build_module"
     done
+    IFS=$IFS_SAVE
```

Comment:
```suggestion
    IFS="$IFS_SAVE"
```
This should be quoted. (I think technically its fine as we have explicitly set the IFS to only space, but its more explicit to quote.)